PG-2110 Update RN with Hotfix Important Note (18)

Andriciuc opened the pull request:
This PR updates the 18.1.1 release note with a hotfix for an issue found in a prior release.

Andriciuc committed:
PG-2110 Update RN with Hotfix Important Note (18)

This PR updates the 18.1.1 release note with a hotfix for an issue found in a prior release.

jeltz commented on docs/release-notes/release-notes-v18.1.1.md:
```diff
@@ -4,13 +4,21 @@
-It introduces several major enhancements, such as: 
+!!! important "PostgreSQL RPMs rebuilt to disable debug assertions"
+
+  Prior to this release, some Percona Server for PostgreSQL RPM packages were inadvertently built with debug assertions enabled (`--enable-cassert`). Debug assertions are intended for **development and testing** only and must **not** be used in production environments.
```
I think saying priori to this release is confusing. Isn't the issue that certain package versions of certain PG versions were affected? I think you should drop "Prior to this release" and ask the build team exactly which package versions were affected. Also this is not just 18.1.1 but all versions listed in Kai's blog post.

jeltz replied:
Also remove this sentence "Debug assertions are intended for **development and testing** only and must **not** be used in production environments.". No need to tell our users this. :rofl:

Andriciuc replied:
Gotcha, thank you for the review Andreas!

Andriciuc replied:
Updated based on packaging team feedback and yours!

Andriciuc committed:
updated based on Andreas's feedback

Andriciuc committed:
Update release-notes-v18.1.1.md

jeltz commented on docs/release-notes/release-notes-v18.1.1.md:
```diff
@@ -4,13 +4,21 @@
-It introduces several major enhancements, such as: 
+!!! important "PostgreSQL RPMs rebuilt to disable debug assertions"
+
+    The Percona Server for PostgreSQL (PSP) and Percona Distribution for PostgreSQL (PPG) RPM packages for **PostgreSQL versions 13 through 18 released as part of the Q4 quarterly release** were built with debug assertions enabled (`--enable-cassert`).
```
The Q3 release was also affect. I think you should just use the list of actual versions from the blog post.

ImTheKai replied:
Yes, please take the exact list, so users can check.

Andriciuc replied:
Updated with proper blog versions for affected packages

Andriciuc committed:
Update release-notes-v18.1.1.md

Andriciuc committed:
Update release-notes-v18.1.1.md

jeltz commented on docs/release-notes/release-notes-v18.1.1.md:
```diff
@@ -4,13 +4,23 @@
-It introduces several major enhancements, such as: 
+!!! important PostgreSQL RPMs rebuilt to disable debug assertions
+
+    The Percona Server for PostgreSQL (PSP) and Percona Distribution for PostgreSQL (PPG) RPM packages for   **PostgreSQL versions 13 through 18 released as part of the Q4 quarterly release** were built with debug assertions enabled (`--enable-cassert`).
```
It was also for the Q3 release.

Andriciuc replied:
Done! Also changed the "you may be affected" mention to "you may suffer performance degradation"

jeltz reviewed (changes requested):
Maybe we should also link to the blog post but that could be something for a separate PR.

Andriciuc committed:
Update release notes for PostgreSQL versions 13-18

replace you may be affected with you may suffer performance degradation and add Q3 to list of affected releases

artemgavrilov commented on docs/release-notes/release-notes-v18.1.1.md:
```diff
@@ -4,13 +4,23 @@
-It introduces several major enhancements, such as: 
+!!! important PostgreSQL RPMs rebuilt to disable debug assertions
+
+    The Percona Server for PostgreSQL (PSP) and Percona Distribution for PostgreSQL (PPG) RPM packages for   **PostgreSQL versions 13 through 18 released as part of the Q3 and Q4 quarterly release** were built with debug assertions enabled (`--enable-cassert`).
```
Do we need 3 spaces here?
```suggestion
    The Percona Server for PostgreSQL (PSP) and Percona Distribution for PostgreSQL (PPG) RPM packages for **PostgreSQL versions 13 through 18 released as part of the Q3 and Q4 quarterly release** were built with debug assertions enabled (`--enable-cassert`).
```

nastena1606 committed:
Update docs/release-notes/release-notes-v18.1.1.md

Co-authored-by: Artem Gavrilov <charlieblackwood7@gmail.com>
